.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
let's name the image `ss-landing-page`. Also it would be better to use a human readable date as tag, e.g. `20240628043835` (to seconds).

Reply:
also make sure you include an additional tag `latest`, (you can achieve this with `docker tag` command) which will be used to automate image pull.